libs/checkout/src/drivers/interfaces/checkout-service.interface.ts

```diff
@@ -1,7 +1,7 @@
 import { Observable } from 'rxjs';
 
-import { Order } from '../../models/order/order';
+import { DaffOrder } from '../../models/order/order';
 
 export interface DaffCheckoutServiceInterface {
```

Comment:
shouldn't this be deprecated?

Reply:
Yeah it should be deprecated. There's a lot of stuff in the checkout package that needs to be changed.